account-abstraction/gas_manager_admin.yaml

```diff
@@ -108,6 +128,10 @@ paths:
             schema:
               type: object
               properties:
+                policyName:
+                  type: string
+                  description: Name of the policy
+                  default: 'My Policy'
```

Comment:
@SahilAujla  this is repeated a few times, should we make it a component?

Reply:
@elanh this is repeated only 2 times. I already tried to make a component out of it but was having some weird issues when I did and referenced it. 

Keeping in mind the current time limitations and other projects, I thought I would come back to this later to debug.